CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-### 1.6.1 (Next)
+### 2.0.0 (2022/07/20)
 * [#111](https://github.com/dblock/iex-ruby-client/pull/111): Added support for fetching symbols for a region - [@mathu97](https://github.com/mathu97).
 * [#111](https://github.com/dblock/iex-ruby-client/pull/111): Added `Symbols#exchange_suffix`, `exchange_name`, `exchange_segment`, `exchange_segment_name`, and `lei` - [@mathu97](https://github.com/mathu97).
-* Your contribution here.
+* [#113](https://github.com/dblock/iex-ruby-client/pull/113): Remove default SSL options for `ca_file` and `ca_path` as well as require Ruby >= 2.4 - [@agrberg](https://github.com/agrberg).
```

Comment:
Either make a separate PR to get us up to Ruby >= 2.4 (or even 2.7), or at least make a separate line like this:

```
[#113](https://github.com/dblock/iex-ruby-client/pull/113): Drop support for Ruby < 2.4 ...
```

Reply:
As `transform_values` was already being used and the update to explicitly require 2.4 was two lines (one in gemspec + one in rubocop) I felt a separate PR wasn't valuable but did pull out the updates in the MD files.